### DISCUSSION TITLE
Alternative beeps

### DISCUSSION THREAD

**Discussion by KuukuuOmoi** [nirgoren/Eddienput#11] [Technical support]
this is so so awesome. is there Any way to set up another beat sound????? having a beep variation would help me lock In loops and standardize route points where I can think to switch it up

**Comment by nirgoren** [comment]
This will be added in a future release.

**Reply by KuukuuOmoi** [reply]
may I ask where in the code you have it "set"  beep as a pull request? if I add "beep2" to the .json and add in a beep2.wav to the directory would the program pull it?

**Reply by nirgoren** [reply]
This is pretty much how it would work in the next version (which will be released soon).
The relevant code is mostly in `src/eddiecontroller.py` lines 112-114 if you want to mess with it.

**Reply by KuukuuOmoi** [reply]
thanks! you're doing such awesome work!

**Reply by nirgoren** [reply]
This is now supported in v1.1.
